nemo/utils/exp_manager.py

```diff
@@ -18,7 +18,7 @@
 import sys
```

Comment:
Can you split the changes here and the ones in timer.py to another PR?

Reply:
done